cli/tests/data/package_build/helloworld/helloworld2.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
They are being used in test_package_add_argument_exclussion, test_service_start_happy_path, and test_service_start_happy_path_json. The reason I decided to create multiple packages instead of using a single package is because we can't uninstall and our system is eventually consistent. This means that using the same package would occasionally cause the tests to fail.